lib/cudnn/nnlib.jl

```diff
@@ -67,6 +94,7 @@ fix1d(cdims::DenseConvDims{1,K,C_in,C_out,S,P,D,F}) where {K,C_in,C_out,S,P,D,F}
 fix1d(pdims::PoolDims{1,K,S,P,D}) where {K,S,P,D,F} =
   PoolDims{2,(K...,1),(S...,1),(P...,0,0),(D...,1)}((pdims.I..., 1), pdims.C_in)
 
+conv_forward = CircularDict{Tuple, Int32}(100)
```

Comment:
Should be `const`

Reply:
fixed in https://github.com/JuliaGPU/CUDA.jl/pull/321/commits/b7d678727d87e75faa56dbe750288b5b1522c35e for all 3 dictionaries